wave_vision/src/utils.cpp

```diff
@@ -27,4 +27,22 @@ void projection_matrix(const Mat3 &K, const Mat3 &R, const Vec3 &t, MatX &P) {
     P = K * extrinsics;
 }
 
+void convertKeypoints(const cv::KeyPoint &keypoint, Vec2 &vec_keypoints) {
+    Vec2 v(keypoint.pt.x, keypoint.pt.y);
```

Comment:
Not necessary, you could have just done
```
vec_keypoints(0) = keypoint.pt.x;
vec_keypoints(1) = keypoint.pt.y;
```

Your version creates an unnecessary temporary variable and it's malloc-ed and free-ed imediately after this function.